cypress/integration/journalistCanLogin.js

```diff
@@ -0,0 +1,74 @@
+/// <reference types="Cypress" />
+
+describe("Journalist can log in", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/articles",
+      response: "fixture:side_articles_shown.json"
+    });
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/articles/**",
+      response: "fixture:article_show.json"
+    });
+  });
+
+  it("unsuccessfully with invalid credentials", () => {
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/v1/auth/sign_in",
+      status: "401",
+      response: {
+        errors: ["Invalid login credentials. Please try again."],
+        success: false
+      }
+    });
+    cy.visit("/");
+    cy.get("#loginButton").click();
+    cy.get("#login").within(() => {
+      cy.get("#email").type("user@mail.com");
+      cy.get("#password").type("wrongpassword");
+      cy.get("button")
+        .contains("Submit")
+        .click();
+    });
```

Comment:
There is no assertion here, should see the response error message?

Reply:
Okay I see what you're doing with the alert but a quick google suggests there is ways to test an alert box with cypress (e.g. https://stackoverflow.com/questions/51795306/how-can-we-test-the-alert-and-text-inside-is-displaying-using-cypress-io-js-auto)

Reply:
I would need to see an assertion though

Reply:
done, changed so that the message handles the error

Reply:
also added assertion